Source/Core/DolphinQt2/DolphinQt2.vcxproj

```diff
@@ -1,4 +1,4 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do not remove VS's BOM as I'm pretty sure it will be added again when saved by VS.